test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -447,6 +447,7 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 
 				gomega.Eventually(func(g gomega.Gomega) {
 					g.Expect(k8sClient.Get(ctx, wlKey, &updatedWl)).To(gomega.Succeed())
+					fmt.Printf("Workload %+v\n", updatedWl)
```

Comment:
@PBundyra Could you remove this debug log?

Reply:
Sure, will do in a follow-up

Reply:
This is needed for cheery picking to 0.8.2

Reply:
taken that: https://github.com/kubernetes-sigs/kueue/pull/3389